### PR TITLE
Porting kubectl-wrapper

### LINKED ISSUE #102: Found more official repo for "kubectl-wrapper"
/kind question

**Question:**
There still one thing for resourceOp, I talked with @vdemeester from tekton, he think the catalog should only maintain the yamls stuff, the source code will make the repo mass to manage.

So we still need find a more official repo.

Another choice is submit the image to a official repo.
@animeshsingh @Tomcli @ckadner @fenglixa

### PR DESCRIPTION
**Pull request by vincent-pli**
**Which issue is resolved by this Pull Request:** 
Resolves #102

### CODE REVIEW

File: tekton-catalog/kubectl-wrapper/README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    `make image TAG=V0.0.1`  
+
+2. Push image to your favourite repo registry  
+
+3. Run `yaml`s in `./deploy` in a `tekton` ready environment, don't forget to replace the image in `kubectl-deploy.yaml`  
+
+## Install the Task
+
+```
+kubectl apply -f https://github.com/vincent-pli/kubectl-wrapper-tekton-step/blob/master/deploy/kubectl-deploy.yaml
```
Comment (Tomcli) [inline tekton-catalog/kubectl-wrapper/README.md:16]: We should update these links to each use this repo or the one in tektoncd/catalog

File: tekton-catalog/kubectl-wrapper/README.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+```
+
+## Install ClusterRolebinding
+
+```
+kubectl apply -f https://github.com/vincent-pli/kubectl-wrapper-tekton-step/blob/master/deploy/rbac.yaml
```
Comment (Tomcli) [inline tekton-catalog/kubectl-wrapper/README.md:22]: same as above

File: tekton-catalog/kubectl-wrapper/Makefile [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+BIN_DIR=_output/bin
+
+# If tag not explicitly set in users default to the git sha.
+TAG ?= v0.0.21
```
Comment (Tomcli) [inline tekton-catalog/kubectl-wrapper/Makefile:18]: Maybe we can set the default to v0.1.0?

File: tekton-catalog/kubectl-wrapper/README.md (hunk 1 above)
Comment (vincent-pli) [reply tekton-catalog/kubectl-wrapper/README.md]: Done

File: tekton-catalog/kubectl-wrapper/README.md (hunk 2 above)
Comment (vincent-pli) [reply tekton-catalog/kubectl-wrapper/README.md]: Done

File: tekton-catalog/kubectl-wrapper/Makefile (hunk 3 above)
Comment (vincent-pli) [reply tekton-catalog/kubectl-wrapper/Makefile]: Done, thanks

**Pull request merged**